### PR TITLE
support AOF

### PR DESCRIPTION
**Pull request by swilly22**

### CODE REVIEW

File: src/commands/cmd_query.c [hunk 1]
```diff
@@ -163,7 +163,9 @@ int MGraph_Query(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
 	CommandCtx *context;
 	int flags = RedisModule_GetContextFlags(ctx);
 	bool is_replicated = RedisModule_GetContextFlags(ctx) & REDISMODULE_CTX_FLAGS_REPLICATED;
-	if(flags & (REDISMODULE_CTX_FLAGS_MULTI | REDISMODULE_CTX_FLAGS_LUA)) {
+	if(flags & (REDISMODULE_CTX_FLAGS_MULTI |
+				REDISMODULE_CTX_FLAGS_LUA |
+				REDISMODULE_CTX_FLAGS_LOADING)) {
```
Comment (jeffreylovitz) [inline src/commands/cmd_query.c]: I had attempted to remove the AOF disabler about a month ago, and encountered errors like:
```
=== REDIS BUG REPORT START: Cut & paste starting from here ===
31443:M 09 Sep 2019 10:52:23.245 # === ASSERTION FAILED ===
31443:M 09 Sep 2019 10:52:23.245 # ==> aof.c:830 '(fakeClient->flags & CLIENT_BLOCKED) == 0' is not true
31443:M 09 Sep 2019 10:52:23.245 # (forcing SIGSEGV to print the bug report.)
31443:M 09 Sep 2019 10:52:23.245 # Redis 5.0.5 crashed by signal: 11
31443:M 09 Sep 2019 10:52:23.245 # Crashed running the instruction at: 0x555c5b785bc0
31443:M 09 Sep 2019 10:52:23.245 # Accessing address: 0xffffffffffffffff
31443:M 09 Sep 2019 10:52:23.245 # Failed assertion: (fakeClient->flags & CLIENT_BLOCKED) == 0 (aof.c:830)
```
Is the `REDISMODULE_CTX_FLAGS_LOADING` flag what solves this problem?

Comment (swilly22) [reply src/commands/cmd_query.c]: It's resolved in the new Redis 5.0.6 release.

**Comment by swilly22** [comment]
@jeffreylovitz, you're right, if the user has both RDB and AOF enabled and the RDB changed node X ID which is later referenced in a query logged by the AOF file, e.g. `MATCH (X) where ID(X) = 2 CREATE (X)-[]->()` then we might be updating the wrong node.

**Comment by gkorland** [comment]
@swilly22 it should be documented that AOF is only supported from Redis 5.0.6

**Pull request merged**